server/src/librelay/relay_server.go

```diff
@@ -355,9 +347,6 @@ func (relay *RelayServer) RegistrationDate() (when int64, err error) {
 	if (iter.Event == nil && !iter.Next()) ||
 		(bytes.Compare(iter.Event.Relay.Bytes(), relay.Address().Bytes()) != 0) ||
 		(iter.Event.TransactionFee.Cmp(relay.Fee) != 0) ||
-		(iter.Event.Stake.Cmp(relay.StakeAmount) < 0) ||
```

Comment:
I'm trying to undertstand what did this line meant...

Reply:
I was validating the values of the staked event but it doesn't really add much and can be removed.